database/migrations/2021_01_21_170224_create_notification_controllers_table.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateNotificationControllersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('notification_controllers', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->unsignedBigInteger('notification_id');
```

Comment:
Unique index for `notification_id` and `controller_position_id`?

Reply:
8d7befa
The generated index name was too long so have replaced it with one of a shorter length